static/index.html

```diff
@@ -49,31 +49,42 @@
       </div>
       <div class="container">
         <div class="row">
-          <div class="col-12 col-sm-12 col-md-12 col-lg-6">
+          <div class="col-12 col-md-12 col-lg-6">
```

Comment:
col-12 col-md-12
col-12 is carried over to bigger browser size, no need to declare again (col-md-12). Basically you will only add another col-class if you want to change number of columns, like what you did for col-lg-6.
You can remove col-md-12 (Please check that it does not break your design.)

Reply:
Done